tools/autograd/derivatives.yaml

```diff
@@ -552,6 +552,7 @@
 
 - name: native_dropout(Tensor input, float p, bool? train) -> (Tensor, Tensor)
   input: "GradMode::is_enabled() ? infinitely_differentiable_native_dropout_backward(grad, result1, (!train.has_value() || !train.value() ? 1 : (p == 1 ? 0.0 : 1.0 / (1.0 - p)))) : native_dropout_backward(grad, result1, (!train.has_value() || !train.value() ? 1 : (p == 1 ? 0.0 : 1.0 / (1.0 - p))))"
+  result0: "(!train.has_value() || train.value()) ? (p == 1 ? 0.0 : 1.0 / (1.0 - p)) * input_t * result1 : at::ones_like(input_t)"
```

Comment:
if train is None or train is True, then we do the dropout.

Otherwise, if train is False, then native_dropout(x) returns x.

Reply:
if native_dropout(x) returns x, would the gradient be input_t?

Reply:
Ah good catch! Fixed.